content/docs/reference/service-urls.md

```diff
@@ -0,0 +1,287 @@
+---
+title: Service URLs
+id: service-urls
+description: This page covers all of Pomerium's service URL settings, including authenticate, authorize, and databroker external and internal service URLs.
+keywords:
+  - authenticate service url
+  - authenticate internal service url
+  - authenticate_service_url
+  - authenticate_internal_service_url
+  - authorize service url
+  - authorize_service_url
+  - authorize internal service url
+  - authorize_internal_service_url
+pagination_prev: null
+pagination_next: null
+toc_max_heading_level: 2
+---
+
+import Tabs from '@theme/Tabs'; import TabItem from '@theme/TabItem'; import PomeriumAuthenticateServices from '@site/content/docs/admonitions/\_pomerium-authenticate-services.mdx';
+
+# Service URL Settings
+
+## Authenticate Service URL
+
+The **Authenticate Service URL** setting defines the externally accessible URL where Pomerium redirects end users (clients) to authenticate against an identity provider.
+
+If Pomerium is running in [split-service mode](/docs/internals/configuration#all-in-one-vs-split-service-mode), each Pomerium service requires the authenticate service URL in its configuration.
+
+:::info
+
+<PomeriumAuthenticateServices />
+
+:::
+
+### How to configure
+
+<Tabs>
+<TabItem value="Core" label="Core">
+
+| **Config file keys** | **Environment variables** | **Type** | **Usage** |
+| :-- | :-- | :-- | :-- |
+| `authenticate_service_url` | `AUTHENTICATE_SERVICE_URL` | `URL` | \***required** |
+
+\* Excluding the `authenticate_service_url` defaults to the [hosted authenticated service](/docs/capabilities/hosted-authenticate-service).
+
+### Examples
+
+```yaml
+authenticate_service_url: https://authenticate.corp.example.com
+```
+
+```bash
+AUTHENTICATE_SERVICE_URL=https://authenticate.corp.example.com
+```
+
+</TabItem>
+<TabItem value="Enterprise" label="Enterprise">
+
+`authenticate_service_url` is a bootstrap configuration setting and is not configurable in the Console.
+
+</TabItem>
+<TabItem value="Kubernetes" label="Kubernetes">
+
+| **[Parameter name](/docs/k8s/reference#spec** | **Type** | **Usage**      |
+| :-------------------------------------------- | :------- | :------------- |
+| `authenticate.url`                            | `URL`    | \***required** |
+
+\* Excluding the `authenticate.url` defaults to the [hosted authenticated service](/docs/capabilities/hosted-authenticate-service).
+
+### Examples
+
+```yaml
+authenticate:
+  url: https://authenticate.corp.example.com
+```
+
+See the [Kubernetes - Global Configuration](/docs/k8s/configure) for more information.
+
+</TabItem>
+</Tabs>
+
+## Authenticate Internal Service URL
+
+The **Authenticate Internal Service URL** setting defines an internally accessible URL where Pomerium redirects end users (clients) to authenticate against an identity provider. Use this setting if Pomerium can't access an external service URL.
```

Comment:
"defines an internally accessible URL where Pomerium redirects end users" — I believe this is not correct. Users will only ever be redirected to the external authenticate service URL. The internal service URLs are for internal communication between Pomerium services.

I think the existing wording was more accurate: "Authenticate Internal Service URL is only required for [split-service mode](https://docs.pomerium.com/docs/internals/configuration#all-in-one-vs-split-service-mode) deployments where Pomerium can’t access the public [Authenticate Service URL](https://docs.pomerium.com/docs/reference/authenticate-service-url)."

Reply:
So, maybe this is something we should clarify for users? I was under the impression that "internal" meant the internal or private network, so basically Pomerium services could communicate with other non-Pomerium services with internal network addresses. Clearly that's wrong, and thanks for the clarification. 

To your second point in the comment above("The other service URL settings are only for split service mode."), would it be helpful to say that the internal settings are only for split-service mode deployments where Pomerium services can't communicate with the external URL?